pkg/sharing/sharing.go

```diff
@@ -155,27 +158,32 @@ func (s *Sharing) CreatePreviewPermissions(inst *instance.Instance) (map[string]
 	doc, _ := permissions.GetForSharePreview(inst, s.SID)
 
 	codes := make(map[string]string, len(s.Members)-1)
+	shortcodes := make(map[string]string, len(s.Members)-1)
 	for i, m := range s.Members {
 		if i == 0 {
 			continue
 		}
 		var err error
 
 		var previousVal string
+		var previousShortCode string
 		var ok bool
 
 		// Checks that we don't already have a sharing code
 		if doc != nil {
 			previousVal, ok = doc.Codes[m.Email]
+			previousShortCode, ok = doc.ShortCodes[m.Email]
```

Comment:
You overwrite the `ok` from the previous line. Are you sure you want to do that?